### PR TITLE
Okta 351253 update react js code samples

### PR DESCRIPTION
**Pull request by KostiantynDrozd-okta**
## Description:
- **What's changed?** 
Updated Okta Auth JS and React code samples to be compatible with Create React Application-4

### Resolves:

* [OKTA-351253](https://oktainc.atlassian.net/browse/OKTA-351253)

### CODE REVIEW

File: packages/@okta/vuepress-site/code/react/okta_react/index.md [hunk 1]
```diff
@@ -69,30 +69,22 @@ Create a `src/SignInForm.jsx` file:
...
-      // pkce: false,
-      issuer: issuer
-      });
     oktaAuth.signIn({ username, password })
     .then(res => {
       const sessionToken = res.sessionToken;
       setSessionToken(sessionToken);
       // sessionToken is a one-use token, so make sure this is only called once
-      authService.redirect({ sessionToken });
+          oktaAuth.signInWithRedirect({ sessionToken });
```
Comment (swiftone) [inline packages/@okta/vuepress-site/code/react/okta_react/index.md:87]: nit: extra indent?

**Review by swiftone — APPROVED** [review]

File: packages/@okta/vuepress-site/code/react/okta_react/index.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ This guide will walk you through integrating authentication into a React app wit
 6. [Connect the Routes](#connect-the-routes)
 7. [Start Your App](#start-your-app)
 
-> This guide is for `@okta/okta-auth-js` v4.0.2 and `@okta/okta-react` v3.0.8.
+> This guide is for `@okta/okta-auth-js` v4.0.2 and `@okta/okta-react` v4.1.0.
```
Comment (shuowu) [inline packages/@okta/vuepress-site/code/react/okta_react/index.md:17]: Let's also update `@okta/okta-auth-js` to the latest version.

File: packages/@okta/vuepress-site/code/react/okta_react/index.md [hunk 3]
```diff
@@ -69,30 +69,22 @@ Create a `src/SignInForm.jsx` file:
...
   const handleSubmit = (e) => {
     e.preventDefault();
-    const oktaAuth = new OktaAuth({
-      // If your app is configured to use the Implicit Flow
-      // instead of the Authorization Code with Proof of Code Key Exchange (PKCE)
-      // you will need to uncomment the below line:
-      // pkce: false,
-      issuer: issuer
-      });
     oktaAuth.signIn({ username, password })
```
Comment (shuowu) [inline packages/@okta/vuepress-site/code/react/okta_react/index.md:82]: use `oktaAuth.signInWithCredentials`

**Comment by shuowu** [comment]
Some comments to use the latest APIs, otherwise LGTM.

**Review by shuowu — APPROVED** [review]

**Comment by KostiantynDrozd-okta** [comment]
Branch closed, because it overlaps with task [OKTA-352475](https://oktainc.atlassian.net/browse/OKTA-352475) and it's already merged to master [pull/1718](https://github.com/okta/okta-developer-docs/pull/1718)

**Pull request closed without merging**